common/on-chip-calib.cpp

```diff
@@ -1151,22 +1151,16 @@ namespace rs2
                 float tmp_y = (get_manager().action == on_chip_calib_manager::RS2_CALIB_ACTION_ON_CHIP_OB_CALIB ? 
                     float(y + 45 + 3 * ImGui::GetTextLineHeightWithSpacing()) : float(y + 40 + 2 * ImGui::GetTextLineHeightWithSpacing()));
                 ImGui::SetCursorScreenPos({ float(x + 9),  tmp_y });
-                if (ImGui::RadioButton("OCC All", (int *)&(get_manager().action), 0))
-                    get_manager().action = on_chip_calib_manager::RS2_CALIB_ACTION_ON_CHIP_OB_CALIB;
-                if (ImGui::IsItemHovered())
-                    ImGui::SetTooltip("%s", "Both OCC Py/Rx calibration and OCC FL calibration in order");
-
-                ImGui::SetCursorScreenPos({ float(x + 6 + width / 3), tmp_y });
-                if (ImGui::RadioButton("OCC Py/Rx", (int*)&(get_manager().action), 1))
+                if (ImGui::RadioButton("OCC", (int*)&(get_manager().action), 1))
                     get_manager().action = on_chip_calib_manager::RS2_CALIB_ACTION_ON_CHIP_CALIB;
                 if (ImGui::IsItemHovered())
-                    ImGui::SetTooltip("%s", "On-chip Py/Rx calibration");
+                    ImGui::SetTooltip("%s", "On-Chip Calibration");
 
-                ImGui::SetCursorScreenPos({ float(x + 3 + 2 * width / 3), tmp_y });
-                if (ImGui::RadioButton("OCC FL", (int*)&(get_manager().action), 2))
-                    get_manager().action = on_chip_calib_manager::RS2_CALIB_ACTION_ON_CHIP_FL_CALIB;
+                ImGui::SetCursorScreenPos({ float(x + 135),  tmp_y });
+                if (ImGui::RadioButton("OCC Extended", (int *)&(get_manager().action), 0))
```

Comment:
Should the default zero-index be assigned to base OCC or extended?

Reply:
default is OCC.
OCC radio button index is 1.